src/bitmessagekivy/tests/sampleData/keys.dat

```diff
@@ -0,0 +1,59 @@
+[bitmessagesettings]
```

Comment:
if I will change .dat extension then it will cause a problem I think

Reply:
Why? There is already [test_pattern/knownnodes.dat](../blob/v0.6/src/tests/test_pattern/knownnodes.dat). Maybe just change the path?

Reply:
Perhaps then no need to rename the files, just a different directory. But I'm not sure all these files are needed (other than perhaps knownnodes.dat).

Reply:
https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/tests/test_pattern/knownnodes.dat is for a different purpose here we are testing older versions of knownnodes.dat file so it will not work in v0.6